components/cmumps2fhir-demographics.js

```diff
@@ -0,0 +1,48 @@
+// cmumps2fhir-demographics.js
+
+var _ = require('underscore');
+
+// NOTE: The rdftransforms are under a private git repository belonging to Hokukahu, LTD.  
+// Please contact Hokukahu if you wish to run this component
+var extractor = require('../../rdftransforms/translate/chcs');
```

Comment:
This can cause a lot of headaches setting up. I think we should use an npm module for rdftransforms if they need to be in a separate repository.